www/tynk00/sp/resources/views/livewire/note-search.blade.php

```diff
@@ -0,0 +1,18 @@
+<div>
+    <div class="p-2 shadow-sm rounded bg-dark my-3">
+        <a href="{{ route('notes.create') }}" class="btn btn-light">Nová poznámka</a>
+        <div class="float-right">
+            <form action="/search-projects" method="get">
+                <input type="text" id="searchInput" class="form-control" wire:model="searchTerm" placeholder="Vyhledat...">
+            </form>
+            
+        </div>
+    </div>
+    <div class="row">
+        @foreach ($notes as $note)
+        @component('components.note-card', ['note' => $note])
```

Comment:
nejde vybrat projekt

Reply:
Mělo by být opraveno na serveru, potom to vložím i sem.